<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLineEdit" name="text_input">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="0">
       <widget class="QPushButton" name="four_btn">
        <property name="styleSheet">
         <string notr="true">border: none;
text-align: center;
text-decoration: none;
display: inline-block;
border-radius: 50%;
background-color: #34495e;
padding: 20px;
color: #ecf0f1;</string>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="two_btn">
        <property name="styleSheet">
         <string notr="true">border: none;
text-align: center;
text-decoration: none;
display: inline-block;
border-radius: 50%;
background-color: #34495e;
padding: 20px;
color: #ecf0f1;</string>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="multiply_btn">
        <property name="styleSheet">
         <string notr="true">border: none;
text-align: center;
text-decoration: none;
display: inline-block;
border-radius: 50%;
background-color: #34495e;
padding: 20px;
color: #ecf0f1;</string>
        </property>
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="eight_btn">
        <property name="styleSheet">
         <string notr="true">border: none;
text-align: center;
text-decoration: none;
display: inline-block;
border-radius: 50%;
background-color: #34495e;
padding: 20px;
color: #ecf0f1;</string>
        </property>
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="six_btn">
        <property name="styleSheet">
         <string notr="true">border: none;
text-align: center;
text-decoration: none;
display: inline-block;
border-radius: 50%;
background-color: #34495e;
padding: 20px;
color: #ecf0f1;</string>
        </property>
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="sign_btn">
        <property name="styleSheet">
         <string notr="true">border: none;
text-align: center;
text-decoration: none;
display: inline-block;
border-radius: 50%;
background-color: #34495e;
padding: 20px;
color: #ecf0f1;</string>
        </property>
        <property name="text">
         <string>+/-</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="one_btn">
        <property name="styleSheet">
         <string notr="true">border: none;
text-align: center;
text-decoration: none;
display: inline-block;
border-radius: 50%;
background-color: #34495e;
padding: 20px;
color: #ecf0f1;</string>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="seven_btn">
        <property name="styleSheet">
         <string notr="true">border: none;
text-align: center;
text-decoration: none;
display: inline-block;
border-radius: 50%;
background-color: #34495e;
padding: 20px;
color: #ecf0f1;</string>
        </property>
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="three_btn">
        <property name="styleSheet">
         <string notr="true">border: none;
text-align: center;
text-decoration: none;
display: inline-block;
border-radius: 50%;
background-color: #34495e;
padding: 20px;
color: #ecf0f1;</string>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="five_btn">
        <property name="styleSheet">
         <string notr="true">border: none;
text-align: center;
text-decoration: none;
display: inline-block;
border-radius: 50%;
background-color: #34495e;
padding: 20px;
color: #ecf0f1;</string>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="nine_btn">
        <property name="styleSheet">
         <string notr="true">border: none;
text-align: center;
text-decoration: none;
display: inline-block;
border-radius: 50%;
background-color: #34495e;
padding: 20px;
color: #ecf0f1;</string>
        </property>
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="substract_btn">
        <property name="styleSheet">
         <string notr="true">border: none;
text-align: center;
text-decoration: none;
display: inline-block;
border-radius: 50%;
background-color: #34495e;
padding: 20px;
color: #ecf0f1;</string>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="dot_btn">
        <property name="styleSheet">
         <string notr="true">border: none;
text-align: center;
text-decoration: none;
display: inline-block;
border-radius: 50%;
background-color: #34495e;
padding: 20px;
color: #ecf0f1;</string>
        </property>
        <property name="text">
         <string>.</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="divide_btn">
        <property name="styleSheet">
         <string notr="true">border: none;
text-align: center;
text-decoration: none;
display: inline-block;
border-radius: 50%;
background-color: #34495e;
padding: 20px;
color: #ecf0f1;</string>
        </property>
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="inverse_btn">
        <property name="styleSheet">
         <string notr="true">border: none;
text-align: center;
text-decoration: none;
display: inline-block;
border-radius: 50%;
background-color: #34495e;
padding: 20px;
color: #ecf0f1;</string>
        </property>
        <property name="text">
         <string>1/x</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="add_btn">
        <property name="styleSheet">
         <string notr="true">border: none;
text-align: center;
text-decoration: none;
display: inline-block;
border-radius: 50%;
background-color: #34495e;
padding: 20px;
color: #ecf0f1;</string>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="sqrt_btn">
        <property name="styleSheet">
         <string notr="true">border: none;
text-align: center;
text-decoration: none;
display: inline-block;
border-radius: 50%;
background-color: #34495e;
padding: 20px;
color: #ecf0f1;</string>
        </property>
        <property name="text">
         <string>sqrt(x)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pow_btn">
        <property name="styleSheet">
         <string notr="true">border: none;
text-align: center;
text-decoration: none;
display: inline-block;
border-radius: 50%;
background-color: #34495e;
padding: 20px;
color: #ecf0f1;</string>
        </property>
        <property name="text">
         <string>x^2</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="zero_btn">
        <property name="styleSheet">
         <string notr="true">border: none;
text-align: center;
text-decoration: none;
display: inline-block;
border-radius: 50%;
background-color: #34495e;
padding: 20px;
color: #ecf0f1;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QPushButton" name="calc_btn">
        <property name="styleSheet">
         <string notr="true">border: none;
text-align: center;
text-decoration: none;
display: inline-block;
border-radius: 50%;
background-color: #34495e;
padding: 20px;
color: #ecf0f1;</string>
        </property>
        <property name="text">
         <string>=</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="clear_btn">
        <property name="styleSheet">
         <string notr="true">border: none;
text-align: center;
text-decoration: none;
display: inline-block;
border-radius: 50%;
background-color: #34495e;
padding: 20px;
color: #ecf0f1;</string>
        </property>
        <property name="text">
         <string>C</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="delete_btn">
        <property name="styleSheet">
         <string notr="true">border: none;
text-align: center;
text-decoration: none;
display: inline-block;
border-radius: 50%;
background-color: #34495e;
padding: 20px;
color: #ecf0f1;</string>
        </property>
        <property name="text">
         <string>Del</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
